CondFormats/JetMETObjects/data/Summer15_V5_MC_JER_AK4PFchs.txt

```diff
@@ -0,0 +1,8 @@
+{1    JetEta     1     JetPt     sqrt([0]^2/x^2+[1]^2/x+[2]^2)    Resolution} 
+-5.0   -3.2      5     10     200    4.5    0.1    0.05
+-3.2   -2.5      5     10     500    4.5    0.1    0.05
+-3.2   -1.3      5     10    1000    4.5    0.1    0.05
+-1.3    1.3      5     10    4000    4.5    0.1    0.05
+ 1.3    2.5      5     10    1000    4.5    0.1    0.05
+ 2.5    3.2      5     10     500    4.5    0.1    0.05
+ 3.2    5.0      5     10     200    4.5    0.1    0.05
```

Comment:
Do you know who I should contact to create a repository on the cms-data org? I tried to find some documentation of the twiki but without any success...